eslint.config.js

```diff
@@ -0,0 +1,54 @@
+import eslintConfigEslint from "eslint-config-eslint";
+import nodeRecommendedModule from "eslint-plugin-n/configs/recommended-module.js";
+import nodeRecommendedScript from "eslint-plugin-n/configs/recommended-script.js";
+import globals from "globals";
+
+export default [
+    {
+        ignores: [
+            "tests/fixtures/",
+            "dist/",
+            "coverage/",
+            "tools/create-test-example.js"
+        ]
+    },
+
+    // eslintConfigEslint[3] is eslint-plugin-n's recommended-script config
+    ...eslintConfigEslint.slice(0, 3),
+    {
+        files: ["**/*.js"],
+        ...nodeRecommendedModule
+    },
+    {
+        files: ["**/*.cjs"],
+        ...nodeRecommendedScript
+    },
+    ...eslintConfigEslint.slice(4),
```

Comment:
This is not very convenient, perhaps `eslint-config-eslint` should export two configurations, one for CommonJS and one for ES Modules?

Reply:
👍 to 3 configs: the `eslint-plugin-n` is not really needed for web projects like eslint.org.

Reply:
Is there a reason we can't just do `...eslintConfigESLint`?

Reply:
> Is there a reason we can't just do `...eslintConfigESLint`?

We can't because it's a configuration for CommonJS modules.

Reply:
Ah, got it.